src/apis/CameraRoll.re

```diff
@@ -20,46 +20,46 @@ external getPhotosParams:
     ~mimeTypes: array(string)=?,
     unit
   ) =>
-  getPhotosParams =
-  "";
+  getPhotosParams;
 
-type photoIdentifier = {
-  .
-  "node": {
-    .
-    "_type": string,
-    "group_name": string,
-    "image": {
-      .
-      "filename": string,
-      "uri": string,
-      "height": float,
-      "width": float,
-      "isStored": Js.Nullable.t(bool),
-      "playableDuration": float,
-    },
-    "timestamp": float,
-    "location":
-      Js.Nullable.t({
-        .
-        "latitude": Js.Nullable.t(float),
-        "longitude": Js.Nullable.t(float),
-        "altitude": Js.Nullable.t(float),
-        "heading": Js.Nullable.t(float),
-        "speed": Js.Nullable.t(float),
-      }),
-  },
+type photoIdentifier = {note}
+and note = {
+  [@bs.as "type"]
+  _type: string,
+  [@bs.as "group_name"]
+  groupName: string,
+  image,
+  timestamp: float,
+  location: Js.Nullable.t(location),
+}
+and image = {
+  filename: string,
+  uri: string,
+  height: float,
+  width: float,
+  isStored: Js.Nullable.t(bool),
+  playableDuration: float,
+}
+and location = {
+  latitude: Js.Nullable.t(float),
+  longitude: Js.Nullable.t(float),
+  altitude: Js.Nullable.t(float),
+  heading: Js.Nullable.t(float),
+  speed: Js.Nullable.t(float),
 };
 
 type photoIdentifiersPage = {
-  .
-  "edges": array(photoIdentifier),
-  "page_info": {
-    .
-    "has_next_page": bool,
-    "start_cursor": Js.Nullable.t(string),
-    "end_cursor": Js.Nullable.t(string),
-  },
+  edges: array(photoIdentifier),
+  [@bs.as "page_info"]
+  pageInfo,
```

Comment:
@MoOx Unless I'm mistaken CameraRoll is no longer part of RN and has been transferred to the community (Flow is set to throw errors when it's called). I think we should remove or at least deprecate these and many other modules for components that have been removed as of 0.60 or 0.61 if not all the way up to 0.62.

We could remove all bindings for components/APIs which themselves have been removed until version x.y (to be determined) add deprecation messages for everything removed since version  x.y.

Regardless of how we handle these, I'll go ahead and prepare bindings for the community package. Since that's a breaking change, I will adopt @Arnarkari93's changes to these files if they are still relevant to the community version.

Reply:
Do you want me to change back to snake_case? Or should I remove this file?

Reply:
No need to change it back or remove the file. I've pushed #668 to remove this among others.

Reply:
@MoOx @Arnarkari93 @cknitt 

I believe the discussion should happen here, as this should be decided for all our bindings.

I have prepared bindings for the community `CameraRoll` package and for the moment, kept the `camelCase` syntax for the sake of consistency as we have followed the convention for Reason so far. Obviously `snake-case` is the convention for OCaml so there's a bit of irony in not following that while it is also the syntax preferred in the package.

Please comment which makes more sense? `camelCase` for consistency or whatever has been chosen in the target package?

Reply:
I think users will expect `camelCase`, so that gets my vote.

Reply:
camelCase LGTM too. We will just need to document that clearly one day :D